CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - **CUMULUS-1740** - `cumulus_meta.workflow_start_time` is now set in Cumulus
   messages
+- **Ability to set custom backend api url in the archive module**
```

Comment:
Super nitpick: can we capitalize api to API in this sentence? And move the notes here about what was added under an `###Added` header?

Reply:
d29a1e3